rnc2rng/parser.py

```diff
@@ -78,6 +79,15 @@ def __repr__(self):
 def start(s, p):
     return Node('ROOT', None, p[0] + p[1])
 
+@pg.production('strlit : LITERAL')
+def primary_literal(s, p): # from datatypeValue
+    return p[0]
+
+@pg.production('strlit : strlit TILDE strlit')
+def primary_concat(s, p):
+    p[0].value = p[0].value + p[2].value
```

Comment:
Doing `+=` here would be slightly nicer, I think, do you agree?

Reply:
For sure!